src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -64,9 +64,7 @@ def _make_causal_mask(
     # ONNX doesn't support `torch.Tensor.triu` properly, thus we use this workaround
     seq_ids = torch.arange(target_length, device=device)
     mask[:, past_key_values_length:] = seq_ids[:, None] < seq_ids[None, :]
-
-    if past_key_values_length > 0:
-        mask[:, :past_key_values_length] = False
+    mask[:, :past_key_values_length] = False
```

Comment:
Can we focus this PR on one thing only, please? The changes to the BLOOM modeling file have no link with the title of the PR.